runhouse/rns/hardware/cluster.py

```diff
@@ -441,10 +410,9 @@ def restart_grpc_server(
         # TODO how do we capture errors if this fails?
         if resync_rh:
             self.sync_runhouse_to_cluster(_install_url=_rh_install_url)
-            self.save_config_to_cluster()
-        kill_proc_cmd = 'pkill -f "python3 -m runhouse.servers.grpc.unary_server"'
-        logfile = f"{self.name}_grpc_server.log"
-        grpc_server_cmd = "python3 -m runhouse.servers.grpc.unary_server"
+        logfile = f"cluster_server_{self.name}.log"
```

Comment:
can you also update the log file reference name in docs/debugging_logging.rst?

Reply:
I can't hit the resolve button hard enough on this one. Maybe I should punchj my screen.